src/main/java/org/prebid/server/auction/ExchangeService.java

```diff
@@ -1103,8 +1123,12 @@ private static ExtBidResponse toExtBidResponse(List<BidderResponse> results, Bid
 
         Integer cacheVideoBidsTtl;
 
+        boolean returnCreativeBids;
+
+        boolean returnCreativeVideoBids;
+
         static BidRequestCacheInfo noCache() {
-            return BidRequestCacheInfo.of(false, false, null, false, null);
+            return BidRequestCacheInfo.of(false, false, null, false, null, true, true);
```

Comment:
i think it doesn't influence but suggest to set `false` value:
```
return BidRequestCacheInfo.of(false, false, null, false, null, false, false);
```

Reply:
looks like it influence
One of tests failed with false values

Reply:
It means there is bug in logic, because we shouldn't analyze `returnCreative` flag if no caching required.